data/locale/en-US.ini

```diff
@@ -93,7 +99,13 @@ NVENC.Preset.Lossless="Lossless"
 NVENC.Preset.LosslessHighPerformance="Lossless High Performance"
 NVENC.RateControl="Rate Control Options"
 NVENC.RateControl.Mode="Mode"
-NVENC.RateControl.Mode.CQP="Constant Quantization Parameter"
+NVENC.RateControl.Mode.Description="Rate control mode selection"
+NVENC.RateControl.Mode.CQP.Description="A flat compression ratio with no regard for bit rates.\nThis yields the highest quality-per-bitrate."
+NVENC.RateControl.Mode.VBR.Description="Sacrifices quality to stay below the upper bitrate limit,\nor saves bitrate where possible."
+NVENC.RateControl.Mode.VBR_HQ.Description="Variable Bitrate with two-pass encoding enabled by default."
+NVENC.RateControl.Mode.CBR.Description="Sacrifices quality in high-motion scenes and performs bit stuffing\nin low-motion scenes to maintain a constant bitrate."
+NVENC.RateControl.Mode.CBR_HQ.Description="Constant Bitrate with two-pass encoding enabled by default."
+NVENC.RateControl.Mode.CBR_LD_HQ.Description="Constant Bitrate optimized for lowest encoding latency, mainly by disabling B-frames and using slice multithreading."
```

Comment:
`, mainly by disabling B-frames and using slice multithreading` does not apply to NVENC unless you know how all NVENC chips work internally.